packages/@aws-cdk/aws-lambda-nodejs/lib/package-manager.ts

```diff
@@ -60,6 +60,7 @@ export class PackageManager {
       os.platform() === 'win32' ? `${runCommand}.cmd` : runCommand,
       ...runArgs,
       bin,
+      ...(this.lockFile === PackageManager.PNPM.lockFile ? ['--'] : []),
```

Comment:
Required to pass parameters to bin command

Reply:
Let's introduce `argsSeparator?: string` in `PackageManager`.